examples/va/nfv/ovs-dpdk-sriov/service-values.yaml

```diff
@@ -26,6 +26,12 @@ data:
       network_vlan_ranges = dpdk1:206:209,dpdk2:206:209,sriov1,sriov2 # CHANGEME
       [oslo_policy]
       policy_file=/etc/neutron/policy.d/policy.yaml
+  nova:
+    template:
+      schedulerServiceTemplate:
+        customServiceConfig: |
+          [filter_scheduler]
+          enabled_filtes = AggregateInstanceExtraSpecsFilter,ComputeFilter,ComputeCapabilitiesFilter,ImagePropertiesFilter,ServerGroupAntiAffinityFilter,ServerGroupAffinityFilter,SameHostFilter,DifferentHostFilter,PciPassthroughFilter,NUMATopologyFilter
```

Comment:
we are just added  AggregateInstanceExtraSpecsFilter by default

https://github.com/openstack-k8s-operators/nova-operator/pull/802


the new default value is 

enabled_filters = AggregateInstanceExtraSpecsFilter,ComputeFilter,ComputeCapabilitiesFilter,ImagePropertiesFilter,ServerGroupAntiAffinityFilter,ServerGroupAffinityFilter,SameHostFilter,DifferentHostFilter,PciPassthroughFilter,NUMATopologyFilter

so this PR should not be needed as this is identical to what is bing added in this commit

Reply:
@Jaganathancse Are you okay with closing this PR given the feedback from Sean?

Reply:
> @Jaganathancse Are you okay with closing this PR given the feedback from Sean?

we can close it.